src/main/webapp/i18n/de/exam.json

```diff
@@ -674,7 +676,13 @@
                     "exerciseTableSingular": "Aufgabe",
                     "points": "Punkte",
                     "numberParticipants": "Teilnehmende",
-                    "variants": "Varianten"
+                    "variants": "Varianten",
+                    "assessmentCheckType": "Check Typ",
```

Comment:
What is that supposed to be. Is that "manual", "automatic" and so on?

Reply:
Not really, it is for:
- Unfinished Assessments
- Unassisted Quiz Exercises
- Unsubmitted Exams